Makefile.am

```diff
@@ -32,7 +32,7 @@ libwd_crypto_la_SOURCES=wd_cipher.c wd_cipher.h wd_cipher_drv.h \
 			wd_dh.c wd_dh.h wd_dh_drv.h \
 			wd_digest.c wd_digest.h wd_digest_drv.h \
 			wd_util.c wd_util.h
-libwd_crypto_la_LIBADD= $(libwd_la_OBJECTS) -ldl
+libwd_crypto_la_LIBADD= $(libwd_la_OBJECTS) -ldl -lnuma
```

Comment:
I remember that it will cause build issue if "-lnuma" is added in "libwd_crypto_la_LIBADD". So you applied a workaround on linking numa library only in test app. Have you fixed the issue?

Reply:
No, add -lnuma at this location can't fixed the issue.  I fixed the issue as follows:
![image](https://user-images.githubusercontent.com/68534547/96118677-6a318d80-0f1e-11eb-8126-92b4f79edd7b.png)